cst438week2_assignment2/src/main/java/cst438_assignment2/domain/CityInfo.java

```diff
@@ -0,0 +1,171 @@
+package cst438_assignment2.domain;
+
+import java.util.Date;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+
+public class CityInfo {
+	
+	int id;
+	String name;
+	String countryCode;
+	String countryName;
+	String district;
+	int population;
+	double temp;
+	String time;
+	
+	public CityInfo() {}
+	
+	public CityInfo(City city, TempAndTime tempAndTime) {
+		this.id = city.getID();
+		this.name = city.getName();
+		this.countryCode = city.getCountry().getCode();
+		this.countryName = city.getCountry().getName();
+		this.district = city.getDistrict();
+		this.population = city.getPopulation();
+		this.temp = tempAndTime.getTemp();
+		this.time = convertTime(tempAndTime);
+	}
```

Comment:
This looks cleaner than my method. Combining both into one constructor instead of what the original file had